pkg/deployer/baremetal/config.go

```diff
@@ -0,0 +1,244 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package baremetal
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"strconv"
+)
+
+// Config is the desired state of a GreptimeDB cluster on bare metal.
+type Config struct {
+	Cluster *Cluster `yaml:"cluster"`
+	Etcd    *Etcd    `yaml:"etcd"`
+}
+
+type Cluster struct {
+	Name     string    `yaml:"name"`
```

Comment:
question: Cluster name double definition, we use `clusterName = args[0]` ?

Reply:
@daviderli614, you're right, the `name` is unnecessary, I will remove it.